README.md

```diff
@@ -21,8 +21,17 @@ composer require escopecz/mautic-form-submit
 ## Usage
 
 ```php
+//basic form data
+$email		= $_REQUEST['email'] ?? $_REQUEST['Email'] ?? $_REQUEST['E-mail'] ?? $_REQUEST['E-MAIL']; // mauticform_input_mkregistraciaslp_email
+$name		  = $_REQUEST['name'] ?? $_REQUEST['Name']; // mauticform_input_mkregistraciaslp_name
+$phone		= $_REQUEST['phone'] ?? $_REQUEST['Phone']; // mauticform_input_mkregistraciaslp_phone
```

Comment:
This is too specific. It shouldn't be in the docs. Why would anyone need to check for 4 different email keys?

Reply:
I tried using this file for various external services, such as website builders.  It turned out that some people are passing parameters in upper case, and others in lower case.  Somewhere it is necessary for third parties (web master, customer marketer) to register a variable manually.  As a result, either we do the most universal and register-independent check of variables, or we take a long time to find out who made a mistake in writing a variable on the transmitting side.